sdk/communication/Azure.Communication.Calling.Server/src/Models/ContentTransferOptions.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.Communication.Calling.Server.Models
+{
+    /// <summary>
+    /// <see cref="ContentTransferOptions"/> is used to provide options for parallel transfers.
+    /// </summary>
+    public struct ContentTransferOptions : IEquatable<ContentTransferOptions>
+    {
+        /// <summary>
+        /// The maximum length of an transfer in bytes.
+        /// </summary>
+        public long? MaximumTransferSize { get; set; }
```

Comment:
what happens if one of these properties is `null` and why are they nullable?

Reply:
`ParallelTranfer` will use the values in the `Constants` file instead.
They are nullable because this allows the user to only chose whatever they want to setup.

Reply:
I moved the defaulting into this class.